sdilene.php

```diff
@@ -95,3 +96,20 @@
     $clanek2,
     $clanek3,
 ];
+
+// definice přístupových údajů
+$admini = [
```

Comment:
Já bych to celé zjenodušil... Nestačí nám jenom heslo? Jedno, a natvrdo? Víc lidí i hesel to celé jen komplikuje a vlastně nic z toho... Nic víc tím imho neukážeme.